src/main/java/seedu/address/model/activity/Amount.java

```diff
@@ -22,6 +22,10 @@ public Amount(double amount) {
         value = amount;
     }
 
+    public double getValue() {
```

Comment:
Value field is already public. Is this necessary :thinking:

Reply:
no clue